### PR TITLE
53 more minimal gaia distro

### PR DESCRIPTION
**Pull request by autra**
This cleans the profile a bit more.

@peppsac please r?

### CODE REVIEW

File: distros/ronin/homescreens.json [hunk 1]
```diff
@@ -1,15 +1,12 @@
 {
   "homescreens": [
     [
-      ["apps", "search"],
```
Comment (peppsac) [inline distros/ronin/homescreens.json:4]: Not sure what is the effect of this removal? (it's the only non-obvious one, so you may want to explain it in the commit message)

Comment (autra) [inline distros/ronin/homescreens.json:4]: actually this should be in the second commit. Correcting that now.

**Comment by autra** [comment]
@peppsac is it clearer now?

**Comment by peppsac** [comment]
Looks good, merging

**Pull request merged**